### PR TITLE
Add Test for GCP Storage Resource Sample

### PR DESCRIPTION
**Pull request by dzou**
This adds a test for the GCP Storage Resource Sample.

Work in progress: #705.

### CODE REVIEW

File: spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample/src/test/java/com/example/GcsApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	@Autowired
+	private TestRestTemplate testRestTemplate;
+
+	@Value("${gcs-resource-test-bucket}")
+	private String bucketName;
+
+	@BeforeClass
+	public static void checkToRun() {
+		assumeThat(
+				"Spanner Google Cloud Storage integration tests are disabled. "
```
Comment (artembilan) [inline spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample/src/test/java/com/example/GcsApplicationTests.java]: I think `Spanner` is not related here.
Just copy/paste artifact 😄

File: spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample/src/main/java/com/example/WebController.java [hunk 2]
```diff
@@ -34,11 +34,12 @@
  * accessed using the Spring Resource Abstraction.
  *
  * @author Mike Eltsufin
+ * @author Daniel Zou
  */
 @RestController
 public class WebController {
 
-	@Value("gs://REPLACE_WITH_YOUR_BUCKET/my-file.txt")
+	@Value("gs://${gcs-resource-test-bucket}/my-file.txt")
```
Comment (meltsufin) [inline spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample/src/main/java/com/example/WebController.java]: nice!

**Comment by dzou** [comment]
Thanks guys, I fixed the copy paste error here and in the other storage integration test :)
PTAL

File: spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample/src/test/java/com/example/GcsApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+		for (Blob blob : blobs.iterateAll()) {
+			blob.delete();
+		}
+	}
+
+	@Test
+	public void testGcsResourceIsLoaded() {
+		BlobId blobId = BlobId.of(bucketName, "my-file.txt");
+		BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("text/plain").build();
+		Blob blob = storage.create(blobInfo, "Good Morning!".getBytes(StandardCharsets.UTF_8));
```
Comment (dmitry-s) [inline spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample/src/test/java/com/example/GcsApplicationTests.java]: seems like you don't use the `blob` variable anywhere after this assignment

Comment (dzou) [reply spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample/src/test/java/com/example/GcsApplicationTests.java]: Nice catch, done.

**Pull request merged**